src/reactpy_django/templatetags/jinja.py

```diff
@@ -0,0 +1,66 @@
+# Copyright © 2023 Innovatie Ltd. All rights reserved.
```

Comment:
All code within this repository must be encompassed by our repository's [copyright licensing](https://github.com/reactive-python/reactpy-django/blob/main/LICENSE.md), as described in the pull request template.

Having your company retain copyright for this one file will not be acceptable. This would mean that legally we would have no right to use, copy, distribute, or modify your contributions.